app/controllers/checkins_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CheckinsController < ApplicationController
+  # before_action :set_checkin, only: [:show, :edit, :update, :destroy]
+  protect_from_forgery unless: -> { request.format.json? }
+
+  # GET /checkins
+  # GET /checkins.json
+  def index
+    @checkins = Checkin.all
+  end
+
+  # POST /checkins
+  # POST /checkins.json
+  def create
+    user_id = User.where(uid: checkin_params["fb_user_id"]).first.id
+    location_id = Location.where(facebook_id: checkin_params["fb_location_id"]).first.id
+    @checkin = Checkin.new(user_id: user_id, location_id: location_id)
```

Comment:
Instead you could do Checkin.new(user: user, location: location) and not use id's above and just use the finds.